test/functional/terminal/tui_spec.lua

```diff
@@ -85,6 +85,23 @@ describe('TUI', function()
     assert_alive()
   end)
 
+  it('works correctly at startup', function()
```

Comment:
"works correctly" is almost always unnecessary in a test description, because tests are usually testing that things work correctly.
```suggestion
  it('resize at startup', function()
```